File: Java Script - Introduction to Programming/9.9. Functions/01. Blank Receipt.js
function blankReceipt() {
    function header() {
        return `CASH RECEIPT\n------------------------------`
    }

    function body() {
        return `Charged to____________________\nReceived by___________________`
    }

    function footer() {
        return `------------------------------\n(c) SoftUni`
    }

    return `${header()}\n${body()}\n${footer()}`
}

// console.log(blankReceipt())

File: Pyrhon - Introduction to Programming/9.9. Functions/01. Blank Receipt.py
def blank_receipt():
        def header():
            return 'CASH RECEIPT\n------------------------------'


        def body():
            return 'Charged to____________________\nReceived by___________________'


        def footer():
            return '------------------------------\n(c) SoftUni'


        return f'{header()}\n{body()}\n{footer()}'


print(blank_receipt())
